Test.cs

```diff
@@ -12,7 +14,12 @@ public static long calculate(long number)
                 throw new Exception("Number must be non-negative.");
             }
```

Comment:
The correction to the recursive call of the `calculate` method is correct, but please ensure the method name is properly capitalized. Change `Calculate(number + 1)` to `Calculate(number - 1)` and also ensure consistent use of the method name `Calculate` if you follow camelCase or PascalCase conventions.